sale_order_line_description/README.rst

```diff
@@ -13,26 +14,34 @@ Configuration
 The user has to belong to group_use_product_description_per_so_line.
 This is possible by selecting the related option in the following menu:
 
-* Settings --> Configuration --> Sale --> Sale Features
+* Settings --> Users --> Select user --> Sale Features
```

Comment:
actually is in `Users --> (select user) -->Technical Settings --> Use only product sale description on order lines` or in `sale --> Settings --> Quotations & Sales -->Allow using only the product sale description on the sales order lines`